src/hupper/cli.py

```diff
@@ -6,13 +6,28 @@
 from .reloader import start_reloader
 
 
+def shutdown_interval_parser(string):
+    """Parses the shutdown interval onto a float > 0.0."""
+    msg = "Shutdown interval must be a float greater than 0"
+    try:
+        value = float(string)
+        if value <= 0.0:
+            raise argparse.ArgumentTypeError(msg)
+        return value
+    except ValueError:
+        raise argparse.ArgumentTypeError(msg)
+
+
 def main():
     parser = argparse.ArgumentParser()
     parser.add_argument("-m", dest="module", required=True)
     parser.add_argument("-w", dest="watch", action="append")
     parser.add_argument("-x", dest="ignore", action="append")
     parser.add_argument("-v", dest="verbose", action='store_true')
     parser.add_argument("-q", dest="quiet", action='store_true')
+    parser.add_argument(
+        "-i", "--shutdown-interval", type=shutdown_interval_parser
```

Comment:
I don't think `-i` should be reserved for this feature just yet. A future `--reload-interval` has as much right to that one as this and it isn't clear and obvious enough to use. Would you mind dropping the `-i`?